cli/webhook.py

```diff
@@ -0,0 +1,27 @@
+import sys
+import slackweb
+
+
+class WebHook:
+
+    def __init__(self):
+        '''this flag makes sure if initialization fails, many hook
+        steps wont try to post message again and again'''
+        self.disabled = True
+        self.webhookURL = ('https://hooks.slack.com/services/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')
+        self.username = 'roger-deploy-bot'
+        self.emoji = ':rocket:'
+        # default channel unless user overrides
+        self.channel = '#testhook'
```

Comment:
If the user does not over ride , should it go to #testhook channel or #rogeros-deploy ?